src/engraving/rendering/score/tdraw.cpp

```diff
@@ -988,18 +988,18 @@ void TDraw::draw(const Box* item, Painter* painter)
 
     const Box::LayoutData* ldata = item->ldata();
 
-    const bool showHighlightedFrame = item->selected() || item->dropTarget();
-    const bool showFrame = showHighlightedFrame || (item->score() ? item->score()->showFrames() : false);
+    const bool showHighlightedBorder = item->selected() || item->dropTarget();
```

Comment:
This and other variables in `TDraw::draw(const Box* item, …)` shouldn't be renamed